app/models/user.js

```diff
@@ -21,5 +21,9 @@ export default Addressable.extend({
 
   fullName: Ember.computed('firstName', 'lastName', function(){
     return (this.get('firstName') + " " + this.get('lastName'));
+  }),
+
+  isOrderFulfilmentUser: Ember.computed('roles', function() {
```

Comment:
If we want to check whether User has some particular role or not we'll have to create computed property for each role... And I think we can refactor it.

May be creating a static function in User model called "hasRole" which takes user object as first argument and role name(string) as second argument and returns truthy value depending or role exists or not.

This should work like rails scopes.

CC: @namrataukirde